tests/internode_encryption/internode_encryption_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright DataStax, Inc.
+// Please see the included license file for details.
+
+package add_racks
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	ginkgo_util "github.com/datastax/cass-operator/mage/ginkgo"
+	"github.com/datastax/cass-operator/mage/kubectl"
+)
+
+var (
+	testName     = "Add racks"
```

Comment:
We need to update the testName and the test namespace name.

Reply:
Thanks, good catch.